frontend/src/components/pages/TeamDashboard/Presentation.js

```diff
@@ -33,9 +33,19 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ReviewLineChart = ({ data, minimum, maximum }) => {
-  // TODO calculate maximum and minimum so that all the graphs have the same proportions
-
+const ReviewLineChart = ({ data, YAxisData }) => {
+  let min = 0;
+  let max = 100;
```

Comment:
these should be const

Reply:
> these should be const

Howzit Viro if I change it to const errors will occur because on lines 47 and 48 I'm reassigning them.